packages/eslint-config-custom/react.js

```diff
@@ -0,0 +1,86 @@
+module.exports = {
+  root: true,
+  env: {
+    browser: true,
+    es2021: true,
+    node: true,
+  },
+  settings: {
+    react: {
+      version: 'detect',
+    },
+  },
+  ignorePatterns: ['dist', '.eslintrc.cjs', 'prettier.config.cjs'],
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:unicorn/recommended',
```

Comment:
@jinmayamashita It is so big plugin. Because it has more than 100 eslint rules, I wonder if there are any rules that conflict with the rules from the official plugins above 🧐